README.md

```diff
@@ -40,6 +40,26 @@ The general outline for changing the spec has 4 steps:
 3.	Find at least one GBFS producer to implement and test the proposed change.
 4.	Submit a final request-for-comments on the proposed change to the issue discussion. If no outstanding issues are identified after one week’s time, and there is general agreement that the proposed change is worthwhile and follows the GBFS guiding principles outlined below, the proposal will be officially adopted.
 
+## Specification Versioning
+To enable the evolution of GBFS, including changes that would otherwise break backwards-compatibility with consuming applications, GBFS documentation is versioned. A git tag in the form of `X.Y` establishes semantic versions. Git tags batch pull requests into versions.
+
+A whole integer increase is used for breaking changes (MAJOR changes). A decimal increase is used for non-breaking changes (MINOR changes or patches).
+
+Examples of breaking changes include:
+
+* Adding or removing a required endpoint or field
+* Changing the data type or semantics of an existing field
+
+Examples of non-breaking changes include:
+ 
+* Adding or removing an optional endpoint or field
+* Adding or removing enum values
+* Modifying documentation or spec language in a way that clarifies semantics or recommended practices
+
+### Version Release Cycles
+* There is no strict limitation on the frequency of MAJOR releases, but the GBFS community aims to limit the MAJOR releases to 2 or less every 12 months. To limit releases, breaking changes can be batched together.
+* MINOR changes may be applied at any time. There is no guideline to limit the number of MINOR changes.
+* GBFS documentation will include a designated long-term support (LTS) branch. The LTS branch must maintain backwards compatibility for a period of 2 years. Non-breaking changes (MINOR) may be applied to the LTS branch.
```

Comment:
I'd suggest that:

`Non-breaking changes (MINOR) may be applied to the LTS branch.`

...be changed to:

`Non-breaking changes (MINOR) will be applied to the LTS branch when relevant.`

In other words, we plan to apply all minor changes to the LTS branch unless there is a specific reason not to.

Reply:
There are a few items that IMHO need to be clarified here:
1. How are the LTS releases chosen/approved? In #15 we said `The current LTS branch would be changed according to the GBFS governance process (currently a vote of the community).`
1. `The LTS branch must maintain backwards compatibility for a period of 2 years` - So I think this means that every 2 years we'd designate a new LTS branch. If so, I'd suggest changing this language to `The LTS branch would maintain its LTS status for at least 2 years, after which a new LTS release and branch would be designated.`, which is similar to the language in #15.

Reply:
I made the suggested changes.